vault/resource_azure_secret_backend_role.go

```diff
@@ -195,6 +202,7 @@ func azureSecretBackendRoleUpdateFields(_ context.Context, d *schema.ResourceDat
 				data[consts.FieldTags] = strings.Join(tags, ",")
 			}
 		}
+		data[consts.FieldPersistApp] = d.Get(consts.FieldPersistApp).(bool)
```

Comment:
Can we also add this param to the `azureSecretBackendRoleRead` function and set the param. Example: https://github.com/hashicorp/terraform-provider-vault/pull/2299/files#diff-8bb2bc55b58871e9ca6bec2eb35ed995f57e91e7aec14d3370ca228b15e3393aR278-R286